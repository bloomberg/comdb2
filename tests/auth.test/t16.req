# OP user
set user 'user1'
set password 'password1'
select '---------- OP user ----------' as user;
create table t1(i int unique)$$
insert into t1 values(1),(2),(3);

# non-OP user
set user 'user2'
set password 'new_password'
select '---------- non-OP user ----------' as user;
analyze t1;

# Run analyze (OP user)
set user 'user1'
set password 'password1'
select '---------- OP user ----------' as user;
analyze t1;
# Cleanup
drop table t1;
