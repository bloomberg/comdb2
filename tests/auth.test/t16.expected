(user='---------- OP user ----------')
(rows inserted=3)
(user='---------- non-OP user ----------')
[analyze t1] failed with rc -3 User does not have OP credentials
(user='---------- OP user ----------')
